test/ServerWide/GetDatabaseRecordTest.ts

```diff
@@ -25,4 +27,25 @@ describe("GetDatabaseRecordTest", function () {
         assert.ok(databaseRecord);
         assert.strictEqual(databaseRecord.databaseName, store.database);
     });
+
+    it("can map types in database record", async () => {
+        await store.maintenance.send(new CreateSampleDataOperation());
+
+        const databaseRecord = await store.maintenance.server.send(new GetDatabaseRecordOperation(store.database));
+
+        const history = databaseRecord.indexesHistory;
```

Comment:
I think there is more keys that are case-sensitive there worth testing.

Reply:
checked in c# - it case-insensitive so I'm matching behavior